README.md

```diff
@@ -415,6 +415,10 @@ const config = {
   // other configurations
   pkce: true,     // required
   dpop: true,
+  dpopOptions: {
+    // skips validition the resulting token response includes `token_type: DPoP`
```

Comment:
```suggestion
    // skips validation to check that the resulting token response includes `token_type: DPoP`
```

Reply:
Can we clarify here if `false` means we skip validation or if `true` does? Right now it reads like `false` skips validation since the example config on the next line is set to `false`